### PR TITLE
Add Jupyter notebook to demonstrate use of GpxWavLabeler

### PR DESCRIPTION
**Pull request by ralatsdc**

### CODE REVIEW

File: DataScience/GpxWavLabeler.ipynb [hunk 1]
```diff
@@ -0,0 +1,356 @@
...
+  {
+   "cell_type": "code",
+   "execution_count": 68,
+   "id": "7a165d91-d274-43ae-ad93-b90e2624ead2",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Assign data home directory, collection filename, sampling filepath, and clip home directory\n",
+    "data_home = str(Path(\"~\").expanduser() / \"Data\" / \"AISonobuoy\")\n",
+    "collection_filename = \"collection.json\"\n",
```
Comment (jbenjoseph) [inline DataScience/GpxWavLabeler.ipynb:24]: What are these json files?

File: DataScience/GpxWavLabeler.ipynb [hunk 2]
```diff
@@ -0,0 +1,356 @@
...
+    "sampling_filepath = str(Path(\"GpxWavLabeler\").parent / \"data\" / \"sampling.json\")\n",
```
Comment (jbenjoseph) [inline DataScience/GpxWavLabeler.ipynb:25]: ditto

File: DataScience/GpxWavLabeler.ipynb [hunk 3]
```diff
@@ -0,0 +1,356 @@
...
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# Export audio with no source present, if it exists\n",
+    "if src_max_stop_t < hyd_min_stop_t:\n",
+    "    gwl.export_audio_interval(\n",
+    "        audio,\n",
+    "        src_max_stop_t,\n",
+    "        hyd_min_stop_t,\n",
+    "        Path(clip_home) / f\"{Path(hydrophone['name'].lower()).stem}-no-source.wav\",\n",
```
Comment (jbenjoseph) [inline DataScience/GpxWavLabeler.ipynb:144]: ```no-source``` is a label? Are there other labelled wav files this notebook creates?

**Review by jbenjoseph — CHANGES_REQUESTED** [review]
Ultimately what we need is a way of creating creating labelled wav files. For examples, files like ```(something)-<label_name>.wav``` or ```<label_name>/*.wav```. If this notebook had a clear example of this it would be super helpful. Thanks Ray!

File: DataScience/GpxWavLabeler.ipynb (hunk 1 above)
Comment (ralatsdc) [reply DataScience/GpxWavLabeler.ipynb:24]: These JSON files do exactly what you asked in the comment above. I will add some additional documentation to make it clearer.

File: DataScience/GpxWavLabeler.ipynb (hunk 2 above)
Comment (ralatsdc) [reply DataScience/GpxWavLabeler.ipynb:25]: As above.

File: DataScience/GpxWavLabeler.ipynb (hunk 3 above)
Comment (ralatsdc) [reply DataScience/GpxWavLabeler.ipynb:144]: Yes. As above, and in documentation to be added.

**Review by jbenjoseph — APPROVED** [review]
Thank you Ray, this documentation is very helpful.

**Pull request merged**